dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-data-validator.test.js

```diff
@@ -0,0 +1,172 @@
+const {
+  validateUkefId,
+  validateExporter,
+  validateBaseCurrency,
+  validateFacilityUtilisation,
+  validateTotalFeesAccrued,
+  validateMonthlyFeesPaid,
+  validatePaymentCurrency,
+  validateExchangeRate,
+} = require('./utilisation-data-validator');
+
+describe('utilisation-data-validator', () => {
+  describe('validateUkefId', () => {
+    it('returns null when a valid UKEF ID is provided', async () => {
+      const validationError = validateUkefId('28374891', 1);
+
+      expect(validationError).toEqual(null);
+    });
+
+    it('returns null when no UKEF ID is provided', async () => {
+      const validationError = validateUkefId(undefined, 1);
+
+      expect(validationError).toEqual(null);
+    });
```

Comment:
Is this one correct? If no facility ID is provided would we not expect to throw an error? Same for the other tests when nothing is provided - I think I understood that Payment Currency could be empty and we would assume base currency.

Reply:
So here I wanted to add in validation to be safe for putting information in the database but I didn't want to include business logic - I don't want the case where DTFS API has stronger validation than portal and they need to be kept in sync. 
E.g. if portal validation passes I don't want DTFS central API to fail and the user journey to fail

Can chuck a comment at the top of the validations file if helpful to communicate that!

Reply:
Makes sense!